### PR TITLE
Adds swab for netherworld critters

### PR DESCRIPTION
**Pull request by tralezab**
### @Krysonism SHOULD GIVE A LOOK AT THIS FIRST!

Adds netherworld critters as a swab. They take primarily maintenance, lavaland, and bartending reagents, they're supplemented by all geyser chems and minorly chemistry, and they are suppressed by any kind of generic growing meat (liquid gibs, nutriment, protein and peptides)

The abberant cells randomly pick a netherworld mob every time they are created. the big jackpot here is migo for the reasonable health and high damage, and the whiff are blank bodies for having good health but awful damage.

## Why It's Good For The Game

Players love migos and other netherworld critters are aight. They're fairly strong, and so have a medium amount of reagents. Tried to get funky with it since they're supposed to be odd creatures.

## Changelog
:cl:
add: New swab for netherworld critters! Make sure you put them into a different vat system than your other ones, as they do not react well with normal reagents!
/:cl:

### CODE REVIEW

File: code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm [hunk 1]
```diff
@@ -374,4 +374,32 @@
...
+						/datum/reagent/consumable/nutriment = -3,
+						/datum/reagent/liquidgibs = -2
+						)
+	virus_suspectibility = 0
+	resulting_atoms = list(/mob/living/simple_animal/hostile/netherworld = 1)
+
+/datum/micro_organism/cell_line/netherworld/succeed_growing(obj/machinery/plumbing/growing_vat/vat)
+	var/random_result = pick(typesof(/mob/living/simple_animal/hostile/netherworld)) //i looked myself, pretty much all of them are reasonably strong and somewhat on the same level. except migo is the jackpot and the faceless is whiff.
+	resulting_atoms = list(random_result = 1)
+	. = ..()
```
Comment (Jared-Fogle) [inline code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:403]:
```suggestion
	return ..()
```

Comment (tralezab) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:403]: sure, why not

File: code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm [hunk 2]
```diff
@@ -374,4 +374,32 @@
...
+						/datum/reagent/consumable/nutriment/protein = -4,
+						/datum/reagent/consumable/nutriment = -3,
+						/datum/reagent/liquidgibs = -2
+						)
+	virus_suspectibility = 0
+	resulting_atoms = list(/mob/living/simple_animal/hostile/netherworld = 1)
+
+/datum/micro_organism/cell_line/netherworld/succeed_growing(obj/machinery/plumbing/growing_vat/vat)
+	var/random_result = pick(typesof(/mob/living/simple_animal/hostile/netherworld)) //i looked myself, pretty much all of them are reasonably strong and somewhat on the same level. except migo is the jackpot and the faceless is whiff.
+	resulting_atoms = list(random_result = 1)
```
Comment (LemonInTheDark) [inline code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:402]: This is bothering travis for some in- TRAVIS I LOVE YOU SO FUCKING MUCH.
image
ya can't use vars in list defs like this
```suggestion
	var/random_result = pick(typesof(/mob/living/simple_animal/hostile/netherworld)) //i looked myself, pretty much all of them are reasonably strong and somewhat on the same level. except migo is the jackpot and the faceless is whiff.
	resulting_atoms = list()
	resulting_atoms[random_result] = 1
```

Comment (tralezab) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:402]: I can't believe travis has done this

Comment (TemporalOroboros) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm]: Apparently you can also do `resulting_atoms = list((random_result) = 1)`

**Comment by ATH1909** [comment]
Maybe we could make blank bodies less of a dud by giving them a special ability? Like, hands or the ability to copy the appearances of humanoids or something?

**Comment by IndieanaJones** [comment]
Also, do you think you could make these obtainable outside of swabbing netherworld mobs, and perhaps then giving an exact pick on what you want when you do swab a netherworld mob?

**Comment by Krysonism** [comment]
Looks good, man. This is going to be very rare though since not very often are people going to be able to obtain both hollow water and a nether creature. 

I don't necessarily think this is a bad thing, since the mechanic gains a lot of more longevity if there are rare mobs you might see that often you can produce.

Because the unusual required & supressive reagents, if we want this to be available as a swab type sample, we should probably put it in new table for paranormal mobs(the eyeball comes to mind) and have them all take hollow water instead of protein as their base required reagent, to create some overlap in their recipes. 

If we put this cell line in a table with regular cell lines, selecting the one you want would be trivial, unless you like contaminate your vat broth with protein.

**Comment by Krysonism** [comment]
> 
> 
> Maybe we could make blank bodies less of a dud by giving them a special ability? Like, hands, being ghost-possessable, or the ability to copy the appearances of humanoids or something?

I think that is a great idea. I believe almost all mobs should have some kind of quirk or ability that makes them deeper than "unga hit bad man".

File: code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm [hunk 3]
```diff
@@ -374,4 +374,33 @@
...
+						/datum/reagent/consumable/nutriment/peptides = -6,
+						/datum/reagent/consumable/nutriment/protein = -4,
+						/datum/reagent/consumable/nutriment = -3,
+						/datum/reagent/liquidgibs = -2
+						)
+	virus_suspectibility = 0
+	resulting_atoms = list(/mob/living/simple_animal/hostile/netherworld = 1)
+
+/datum/micro_organism/cell_line/netherworld/succeed_growing(obj/machinery/plumbing/growing_vat/vat)
+	var/random_result = pick(typesof(/mob/living/simple_animal/hostile/netherworld)) //i looked myself, pretty much all of them are reasonably strong and somewhat on the same level. except migo is the jackpot and the blank body is whiff.
```
Comment (Qustinnus) [inline code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:401]: is the basetype a mob you want spawned or?

Comment (tralezab) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:401]: yep! it's the weird creature thing in the wizard den. works nice!

**Pull request merged**